operator_ui/src/pages/Bridges/Show.test.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import { syncFetch } from 'test-helpers/syncFetch'
+import { mountWithProviders } from 'test-helpers/mountWithTheme'
+import { Route } from 'react-router-dom'
+import * as ReactRouterDom from 'react-router-dom'
+import { Show } from 'pages/Bridges/Show'
+import globPath from 'test-helpers/globPath'
+
+const mockHistoryPush = jest.fn()
+jest.mock('react-router-dom', () => ({
+  ...(jest.requireActual('react-router-dom') as typeof ReactRouterDom),
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}))
```

Comment:
Why do you need to mock it? Because we are using a memory router in `mountWithProviders` it should be perfectly fine to simply inspect the location without having to mock `react-router-dom`

Reply:
Great! Thanks for the tip

I've removed the mock and added the location check